lib/udap_security_test_kit/endpoints/mock_udap_server.rb

```diff
@@ -20,44 +21,33 @@ def udap_server_metadata(suite_id)
         udap_authorization_extensions_required: [],
         udap_certifications_supported: [],
         udap_certifications_required: [],
-        grant_types_supported: ['client_credentials'],
+        grant_types_supported: ['authorization_code', 'client_credentials'],
```

Comment:
```suggestion
        grant_types_supported: ['authorization_code', 'client_credentials', `refresh_token`],
```
Since you included support for token refresh (which is optional BTW) it should be reflected in the server metadata, even if there are no explicit tests for it.